pages/travel-and-leave/travel-and-leave-policies/first-time-travel-get-in-concur-pre-olu.md

```diff
@@ -1,21 +1,27 @@
 ---
-title:
-  First-time travel guide - Step 1 - Get set up to travel for TTS
-  <em>before</em> GSA onboarding
+title: Get set up to travel for TTS <em>before</em> GSA onboarding
```

Comment:
Based on the content of the page, I think it's not really part of step 1a - it's an alternative to getting Concur access. My thinking is that it shouldn't have the counter step (1a, get concur access) activated, even if we do keep it under that "first time travel" step.

Reply:
Cool, I got rid of the step counter on the page.